packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -161,13 +161,14 @@ export default (async function runDoctor(
     });
   }
 
-  const onKeyPress = async key => {
-    // $FlowFixMe
+  const onKeyPress = async (key: string) => {
+    // @ts-ignore
     process.stdin.setRawMode(false);
```

Comment:
Does `stdin.setRawMode` really exist? It's not in the typings.

Reply:
It exists in TTY, which is our primary use case, so yep: https://nodejs.org/api/tty.html#tty_readstream_setrawmode_mode
Please check if `typeof process.stdin.setRawMode === "function"` before accessing it. We should add it to every call of this method at some point.

Reply:
My bad, it's optional. Thanks!